content/applications/general/iot/config/troubleshooting.rst

```diff
@@ -343,6 +343,66 @@ Save`.
              :align: center
              :alt: Receipt format using naming convention: EpsonTMU220B__IMC_LDV_LDH_SCALE35__.
 
+DYMO LabelWriter print delay
+----------------------------
+
+The DYMO LabelWriter (450 DUO) has a known issue in printing with the :abbr:`IoT (Internet of
+Things)` box while using the label function. To resolve this issue, the printer must be reinstalled.
+First, open the OpenPrinting CUPS administration page from the :abbr:`IoT (Internet of Things)` box
```

Comment:
```suggestion
First, open the :guilabel:`OpenPrinting CUPS` administration page from the :abbr:`IoT (Internet of Things)` box
```
if this is how it appears on the interface, use this GUI -- if not, feel free to ignore this suggestion